azure-pipelines.yml

```diff
@@ -18,17 +18,32 @@ jobs:
         publishJUnitResults: true
         testResultsFiles: '**/TEST-*.xml'
 
-  # Run all tests when running the Windows CI tests
+  # Run all core tests when running the Windows CI tests
   - task: Gradle@2
     condition: eq(variables['Agent.OS'], 'Windows_NT')
-    displayName: Build & test (Windows - all modules)
+    displayName: Build & test (Windows - core)
     env:
       AWS_ACCESS_KEY_ID: $(aws.accessKeyId)
       AWS_SECRET_ACCESS_KEY: $(aws.secretAccessKey)
     inputs:
         gradleWrapperFile: 'gradlew'
         jdkVersionOption: '1.11'
         options: '--no-daemon --continue'
-        tasks: 'clean check'
+        tasks: 'clean testcontainers:check'
+        publishJUnitResults: true
+        testResultsFiles: '**/TEST-*.xml'
+
+  # Run all non-core tests when running the Windows CI tests
+  - task: Gradle@2
+    condition: eq(variables['Agent.OS'], 'Windows_NT')
+    displayName: Build & test (Windows - all non-core modules)
+    env:
+      AWS_ACCESS_KEY_ID: $(aws.accessKeyId)
+      AWS_SECRET_ACCESS_KEY: $(aws.secretAccessKey)
+    inputs:
+        gradleWrapperFile: 'gradlew'
+        jdkVersionOption: '1.11'
+        options: '--no-daemon --continue'
+        tasks: 'clean check -x testcontainers:test'
```

Comment:
Is `testcontainers:test` intentional? I would've expected `testcontainers:check`.